contrib/Dockerfile.systemd.test

```diff
@@ -0,0 +1,142 @@
+# This dockerfile is used to test containerd within a container
+#
+# usage:
+# 1.) docker build -t containerd-test -f Dockerfile.test ../
+# 2.) docker run -it --privileged -v /tmp:/tmp --tmpfs /var/lib/containerd-test containerd-test  bash
+# 3.) $ make binaries install test
+#
+# Use the RUNC_VERSION build-arg to build with a custom version of runc, for example,
+# to build runc v1.0.0-rc94, use:
+#
+# docker build -t containerd-test --build-arg RUNC_VERSION=v1.0.0-rc94 -f Dockerfile.test ../
+
+# ------------------------------------------------------------------------------
+# Public stages:
+# "integration": for running integration tests:
+#   docker build -t containerd-test -f Dockerfile.test --target integration ../
+#   docker run --privileged containerd-test
+#
+# "cri-integration": for running cri-integration tests:
+#   docker build -t containerd-test -f Dockerfile.test --target cri-integration ../
+#   docker run --privileged --sysctl net.ipv6.conf.all.disable_ipv6=0 containerd-test
+#
+# "critest: for running critest:
+#   docker build -t containerd-test -f Dockerfile.test --target critest ../
+#   docker run --privileged --sysctl net.ipv6.conf.all.disable_ipv6=0 containerd-test
+#
+# "cri-in-userns": for running critest with "CRI-in-UserNS" mode; needs Rootless Docker/Podman/nerdctl:
+#   docker build -t containerd-test -f Dockerfile.test --target cri-in-userns ../
+#   docker run --privileged containerd-test
+# ------------------------------------------------------------------------------
+
+ARG GOLANG_VERSION=1.21.3
```

Comment:
- This should probably be 1.21.5 (assuming https://github.com/containerd/containerd/pull/9351 gets merged before this PR is reviewed)

Reply:
thanks, when this pr https://github.com/containerd/containerd/pull/9351 merged after i can update it.